UI/CwCheatScreen.cpp

```diff
@@ -40,10 +41,69 @@ static bool enableAll = false;
 static std::vector<std::string> cheatList;
 static CWCheatEngine *cheatEngine2;
 static std::deque<bool> bEnableCheat;
+std::string gamePath_;
```

Comment:
This should be a class member of `CwCheatScreen` probably, or otherwise it should be `static`.

For non-class members, `static` means that other cpp files can't see it (e.g. if two files had `std::string gamePath_` in them without `static`, they would conflict, or be the same variable.)

-[Unknown]